eox_nelp/pearson_vue/pipeline.py

```diff
@@ -16,10 +18,43 @@
 from eox_nelp.edxapp_wrapper.student import anonymous_id_for_user
 from eox_nelp.pearson_vue.constants import PAYLOAD_CDD, PAYLOAD_EAD, PAYLOAD_PING_DATABASE
 from eox_nelp.pearson_vue.utils import update_xml_with_dict
+from eox_nelp.signals.utils import get_completed_and_graded
+
+logger = logging.getLogger(__name__)
 
 User = get_user_model()
 
 
+def terminate_not_full_completion_cases(user_id, course_id, **kwargs):
+    """Pipeline that check the case of completion cases on the pipeline execution. Also this pipe
+    has 3 behaviours depending the case:
+        - is_passing is true means the course is graded(passed) and dont needs this pipe validation.
+          The pipeline continues without changes.
+        - is_complete=True and is_graded=False pipeline should continue with uploaded metadata
+          (completed courses and not graded).
+        - Otherwise the pipeline will stop, for grading-courses the COURSE_GRADE_NOW_PASSED signal would act.
```

Comment:
```suggestion
        - Otherwise this indicates that the pipeline execution could be stopped, for grading-courses the COURSE_GRADE_NOW_PASSED signal would act.
```

??